### ISSUE TITLE
[Bug Report] autocomplete / combobox inconsistent highlight of selected option

### ISSUE THREAD

**Issue by bviala** [vuetifyjs/vuetify#5706]
### Versions and Environment
**Vuetify:** 1.3.9
**Vue:** 2.5.17
**Browsers:** Chrome 70.0.3538.102
**OS:** Mac OS 10.14.1

### Steps to reproduce
Open the autocomplete and select Programming.
Open the autocomplete again and clear the input, then add the letter p.
You will notice that Programming option is not highlighted when it comes in the first option slot.
Remove the letter p. 
You will notice that both Gaming and Programming option are highlighted for a short instant, before Gaming fade out.

### Expected Behavior
Selected option remains highlighted when option list changes.
Not selected options remains not highlighted when option list changes.

### Actual Behavior
Inconsistent highlighting

### Reproduction Link
<a href="https://codepen.io/bviala/pen/aQYdzb?editors=1010" target="_blank">https://codepen.io/bviala/pen/aQYdzb?editors=1010</a>

### Other comments
v-select probably has the same issue, I didn't test.
Maybe remove the fading effect for this, it's not very important.

**Comment by johnleider** [comment]
Need a confirmation if this is browser based. Cannot reproduce on W10/Chrome

**Comment by nekosaur** [comment]
I am not able to reproduce it on Mac OS 10.14.1 with Chrome 71.0.3578.98. But I am a bit unsure about the reproduction steps. When in step 2 you say `clear the input`, does that mean pressing and holding backspace, or selecting all text and pressing backspace/delete?

**Comment by bviala** [comment]
@johnleider @nekosaur 
I confirm it's not browser based. I can reproduce on Chrome 71.0.3578.98, Safari 12.0.2 and Firefox 64.0, although in firefox it's almost unnoticeable, because for a reason the highlight fading is way faster.  

I'll try to give clearer instruction:

1. Click on the autocomplete. The options list appears.
2. Click on Gaming to select it. The options list closes.
3. Click again on the autocomplete to reopen it. The autocomplete text field contains "Gaming". Clear the text field by hitting backspace.
4. The first item in the option list is highlighted, which is fine, because it is "Gaming", the actual selected value
5. Now, type a "p" in the autocomplete. Only the option "Programming" remains in the option list.
6. For a short instant, the first item in the option list stays highlighted, which is not fine, because it is "Programming", which is not the selected value.
7. After that, the first and now only option of the list becomes not highlighted, which is fine, because it is "Programming", which is not the selected value.
8. Now, hit backspace to clear the autocomplete textfield again. All the options now appear in the list.
9. For a short instant, the first item in the option list stays not highlighted, which is not fine, because it is "Gaming", the actual selected value.
10. Repeat (type p, observe, remove p, observe etc.)

In short terms, the highlighted property seem to be base on the position of the selected item in the option list, which causes problem because of:

- Fading
- The items position can change

It's quite a minor bug, but still.

**Comment by nekosaur** [comment]
Thanks for the detailed reproduction 👍 I can see the issue now.

**Comment by trentmurray** [comment]
I have just tested on version 2.0.0 and this is no longer an issue. 

@nekosaur Is there still a need to fix this in versions < 2?

**Comment by johnleider** [comment]
This will not be resolved for v1.5.
